astropy/cosmology/core.py

```diff
@@ -98,6 +260,11 @@ def name(self):
         """The name of the Cosmology instance."""
         return self._name
 
+    @property
+    def parameters(self):
```

Comment:
This looks rather like `.info()`?!

Reply:
It does! I've been meaning to make a ``.info()`` attribute. Maybe here would be the place for a minimal implementation.
It couldn't be the Column mixin; I've tried, and the machinery is not (yet) sophisticated enough for non-broadcastable non-numeric scalar objects like ``Cosmology``.
When I get the ``to/from_format("astropy.table")`` PR ready, then ``.info`` can also be used to return a Table form of the cosmology.

Reply:
I've implemented a barebones ``info``. To be expanded upon in future PRs.